### ISSUE TITLE
Regarding the training and testing sets

### ISSUE THREAD

**Issue by CyberFreshMan** [chenning0115/SpectralDiff#4]
Hello, in the spectral-spatial diffusion module you provided, the data code does not include splitting the data into training and testing sets. May I know how you perform the data splitting?

**Comment by skylzh14** [comment]
Hello，I also want to know how to split the training set and test set,could you provide your method?
Thank you very much!

**Comment by chenning0115** [comment]
Thanks for concern about our work. we just random split the data into traning and test dataset.
